heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraCLIController.java

```diff
@@ -111,29 +113,43 @@ public void removeContainers(Set<PackingPlan.ContainerPlan> containersToRemove)
   }
 
   @Override
-  public void addContainers(Integer count) {
+  public Set<Integer> addContainers(Integer count) {
```

Comment:
Should we return a list to preserve order or set?

Reply:
i prefer set because aurora returns unique ids. aurora does not enforce id order

Reply:
But don't you want a list of ids with increasing order? 

It could potentially help the remapping with less items.

Reply:
e. how can it help remapping with less items